lib/sensu-plugins-memory-checks/version.rb

```diff
@@ -2,7 +2,7 @@ module SensuPluginsMemoryChecks
   module Version
     MAJOR = 3
     MINOR = 0
-    PATCH = 2
+    PATCH = 3
```

Comment:
please leave versioning to the maintainers, this reduces the number of times we have to ask contributors to sync and rebase their fork.